test_scripts/API/VehicleData/StabilityControlsStatus/015_SubscribeVehicleData_StabilityControlsStatus_invalid_hmi_response.lua

```diff
@@ -0,0 +1,84 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check subscription on StabilityControlsStatus data via SubscribeVehicleData RPC
+--   with invalid HMI response
+--
+-- Precondition:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- 3) PTU is successfully performed
+-- 4) App is activated
+
+-- Steps:
+-- 1) App sends SubscribeVehicleData request with (stabilityControlsStatus = true) to SDL
+-- SDL does:
+--  - send VehicleInfo.SubscribeVehicleData request with (stabilityControlsStatus = true) to HMI
+-- HMI sends invalid VehicleInfo.SubscribeVehicleData response related to stabilityControlsStatus
+-- SDL does:
+--  - send SubscribeVehicleData response with (success: false, resultCode: "GENERIC_ERROR")
+-- 2) HMI sends VehicleInfo.OnVehicleData notifications with StabilityControlsStatus data
+-- SDL does:
+--  - not send OnVehicleData notification with received from HMI data to App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/API/VehicleData/commonVehicleData')
+
+--[[ Local variables ]]
+local vdNameInvalidRes = "stabilityControlsStatus"
+local mobileRequest = {
+    [vdNameInvalidRes] = true
+  }
+local hmiRequest = mobileRequest
+
+local hmiResponseEmpty = {
+    [vdNameInvalidRes] = {},
+  }
+local hmiResponseInvalidValues = {
+    [vdNameInvalidRes] = { resultCode = "SUCCESS", dataType = "xxx" },
```

Comment:
```suggestion
    [vdNameInvalidRes] = { resultCode = "SUCCESS", dataType = "invalidType" },
```
I would propose to rename value of dataType

Reply:
@VjKlepikov please check https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/15/commits/e86562fbe628dad0e5104d3c6551760077385116